test/e2e/benchmark/benchmark.go

```diff
@@ -157,9 +159,8 @@ func (b *BenchmarkTest) CheckResults() error {
 	}
 
 	// check if any tx has been submitted
-	log.Println("Reading blockchain")
-	blockchain, err := testnode.ReadBlockHeights(context.Background(),
-		b.Node(0).AddressRPC(), 1, 5)
```

Comment:
Before, the chain was only read until height `5`. As the `txsim` instances start after the validators, the first blocks do not contain any txs. Now, we are reading the whole blockchain.

Reply:
Thanks Samuel, it was intentional to reduce the height to 5 as otherwise it would take a long time to read the blockchain specially for large size blocks, and would fail from one run to another. But if it works for the entire blockchain, that would be even better, thanks!